### PR TITLE
Allow admins to force markings onto IPCs

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**

### CODE REVIEW

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/silicon_base.yml [hunk 1]
```diff
@@ -49,6 +49,55 @@
...
+        - map: ["enum.HumanoidVisualLayers.FacialHair"]
+        - map: ["enum.HumanoidVisualLayers.Hair"]
+        - map: ["enum.HumanoidVisualLayers.HeadSide"]
+        - map: ["enum.HumanoidVisualLayers.HeadTop"]
+        - map: ["mask"]
+        - map: ["head"]
+        - map: ["pocket1"]
+        - map: ["pocket2"]
+        - map: ["enum.HumanoidVisualLayers.Tail"]
+        - map: ["enum.HumanoidVisualLayers.Wings"]
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/silicon_base.yml]: What's this all about? The non-humanoid robot base specifically doesn't have all this.

File: Resources/Prototypes/SimpleStation14/Species/ipc.yml [hunk 2]
```diff
@@ -28,6 +28,11 @@
   id: MobIPCSprites
   sprites:
     Head: MobIPCHead
+    Snout: MobHumanoidAnyMarking
+    HeadTop: MobHumanoidAnyMarking
+    HeadSide: MobHumanoidAnyMarking
+    Wings: MobHumanoidAnyMarking
+    Tail: MobHumanoidAnyMarking
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Species/ipc.yml:35]: So this essentially gives them the 'slots' for these markings, but without the white list set they can only be forced on. I mean like, sure I guess, seems rather unneeded to me.
One concern I have is will this cause these marking categories to be selectable in the marking editor even though they're all empty?

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/SimpleStation14/Species/ipc.yml:35]: The player-facing marking editor is unchanged, this just *allows* the markings the potential to be added if other things agree. If you want the players to add them in character customization you'd need to edit their marking points prototype or whatever it is

Comment (Pspritechologist) [reply Resources/Prototypes/SimpleStation14/Species/ipc.yml:35]: Sounds good.

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**